ext/Rakefile

```diff
@@ -24,7 +23,7 @@ def create_task_for_unix(config)
 
   tarball = config[:tarball]
   target  = config[:target]
-  url = "#{AMAZON}/#{tarball}"
+  url = config[:url]
```

Comment:
The Weyback machine is a mostly trustworthy site, but it would be good to allow this variable to be specified externally for skeptical users. For example:

```ruby
url = ENV['KINDLEGEN_TARBALL_URL'] || config[:url]
```